src/pytezos/cli/cli.py

```diff
@@ -22,6 +28,8 @@
     "codemirror_mode": "michelson",
 }
 
+SANDBOX_IMAGE = 'bakingbad/sandboxed-node:v9.0-rc1-1'
```

Comment:
User may want to use it's own image, it's better to make this a CLI option

Reply:
You mean just the tag or the image name too?

Reply:
Both image and tag.

Reply:
Done